pipeline/04_hybrid_rag.py

```
@@ -266,21 +272,31 @@ def hybrid_rag(query, use_vector=True, use_graph=True, verbose=True):
-        vector_docs = vector_search(query, n_results=3)
-        if verbose:
-            print(f"\n[Vector 검색] {len(vector_docs)}개 문서 검색됨")
-            for doc in vector_docs:
-                print(f"  - {doc['source']} (유사도: {doc['score']})")
+        try:
+            vector_docs = vector_search(query, n_results=3)
+            if verbose:
+                print(f"\n[Vector 검색] {len(vector_docs)}개 문서 검색됨")
+                for doc in vector_docs:
+                    print(f"  - {doc['source']} (유사도: {doc['score']})")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)